app-react/src/main/kotlin/com/ustadmobile/view/ReportFilterEditScreen.kt

```diff
@@ -0,0 +1,201 @@
+                            props.uiState.reportFilter?.shallowCopy {
+                                reportFilterDropDownValue = it?.value ?: 0
+                            })
+                    }
+                }
+            }
+
+            UstadTextEditField {
+                value = props.uiState.reportFilter?.reportFilterValue ?: ""
+                label = strings[MessageID.report_filter_edit_values]
+                error = props.uiState.valuesError
+                enabled = props.uiState.fieldsEnabled
+                inputProps = {
+                    it.inputMode = InputMode.numeric
+                }
+                onChange = {
+                    props.onReportFilterChanged(
+                        props.uiState.reportFilter?.shallowCopy {
+                            reportFilterValue = it
+                        })
+                }
+            }
+
+            Stack {
+                direction = responsive(StackDirection.row)
+                spacing = responsive(10.px)
+
+                UstadTextEditField {
+                    value = props.uiState.reportFilter?.reportFilterValueBetweenX ?: ""
+                    label = strings[MessageID.from]
+                    error = props.uiState.valuesError
+                    enabled = props.uiState.fieldsEnabled
+                    inputProps = {
+                        it.inputMode = InputMode.numeric
+                    }
+                    onChange = {
+                        props.onReportFilterChanged(
+                            props.uiState.reportFilter?.shallowCopy {
+                                reportFilterValueBetweenX = it
+                            })
+                    }
+                }
+
+                UstadTextEditField {
+                    value = props.uiState.reportFilter?.reportFilterValueBetweenY ?: ""
+                    label = strings[MessageID.toC]
+                    error = props.uiState.valuesError
+                    enabled = props.uiState.fieldsEnabled
+                    inputProps = {
+                        it.inputMode = InputMode.numeric
+                    }
+                    onChange = {
+                        props.onReportFilterChanged(
+                            props.uiState.reportFilter?.shallowCopy {
+                                reportFilterValueBetweenY = it
+                            })
+                    }
+                }
+            }
+
+            List{
+                props.uiState.uidAndLabelList.forEach { uidAndLabel ->
+                    ListItem{
+                        ListItemText {
+                            primary = ReactNode(uidAndLabel.labelName ?: "")
+                            onClick = {
+                                props.onClickEditFilter(uidAndLabel)
+                            }
+                        }
+                        secondaryAction = IconButton.create {
+                            Delete {}
```

Comment:
Please add ariaLabel property so that this button is accessible (same as contentDescription on Jetpack)
e.g.
```
IconButton.create {
    ariaLabel = strings[MessageID.delete]
    onClick = {  
       ...
    }
    Delete { } 
}
```